theme-settings.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file
+ * SAA Theme Settings.
+ */
```

Comment:
We could add one line saying

Note: Theme settings are close to Stanford Basic but hide brandbar and masthead style options, since those are enforced with custom styles in this theme.